shared/chat/conversation/info-panel/add-people.tsx

```diff
@@ -92,7 +92,11 @@ const AddPeople = compose(
 )(_AddPeople) as any
 
 const styles = Styles.styleSheetCreate({
-  addButtonContainer: {marginLeft: Styles.globalMargins.small, marginRight: Styles.globalMargins.small},
+  addButtonContainer: {
+    marginBottom: Styles.globalMargins.small,
```

Comment:
cc @mmaxim do any height calculations need to change because of this?

Reply:
@mmaxim ^?

Reply:
Don't think so.